### ISSUE TITLE
How should I use Normalize_TRT plugin？

### ISSUE THREAD

**Issue by ye1024** [NVIDIA/TensorRT#1016]
I want to know about the algorithm of Normalize_TRT plugin  is Batch_normalization or Layer_normalization, 
and how to use it
thanks！

**Comment by ttyio** [comment]
Hello @ye1024 ,  thanks for question.
From the readme https://github.com/NVIDIA/TensorRT/tree/master/plugin/normalizePlugin, I see this plugin is used in parseNet and SSD.  It has field `acrossSpatial` to switch between batch and layernormalization.  
Beside this plugin, we also support onnx operator `BatchNormalization`, it is implemented using TRT `pointwise` API see code in https://github.com/onnx/onnx-tensorrt/blob/acb08055e71d03daa40e3e67e1b99eda0eafcf43/builtin_op_importers.cpp#L232
We also have a layernorm + residual fused plugin specified for BERT in https://github.com/NVIDIA/TensorRT/tree/master/plugin/skipLayerNormPlugin

**Comment by ttyio** [comment]
Closing since no activity in this thread for more than 3 weeks, please reopen if you still have question, thanks!
